src/storage/src/hummock/sstable/bloom.rs

```diff
@@ -156,4 +158,44 @@ mod tests {
         assert!(f.surely_not_have_hash(check_hash[2]));
         assert!(f.surely_not_have_hash(check_hash[3]));
     }
+
+    fn false_positive_rate_case(
+        preset_key_count: usize,
+        test_key_count: usize,
+        expected_false_positive_rate: f64,
+    ) {
+        let mut key_list = vec![];
+
+        for i in 0..preset_key_count {
+            let k = Bytes::from(format!("{:032}", i));
+            let h = farmhash::fingerprint32(&k);
```

Comment:
can we unify the hash function to `xxh` ? like sstable::builder

Reply:
done